src/main/java/com/coreos/jetcd/ClientBuilder.java

```diff
@@ -14,13 +14,17 @@
 import io.grpc.NameResolver;
 import io.grpc.netty.GrpcSslContexts;
 import io.netty.handler.ssl.SslContext;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Arrays;
 import java.util.List;
 
 /**
  * ClientBuilder knows how to create an Client instance.
  */
 public class ClientBuilder implements Cloneable {
-
+  private static final List<String> SCHEMES = Arrays.asList("http", "https", "unix", "unixs");
```

Comment:
@lburgazzoli for now, I think we should just accept `http/https` and get this pr merged. we can refactor the check into `NameResolver ` in a different pr after https://github.com/coreos/jetcd/issues/176 is discussed. let me know what you think.